app/views/layouts/mail/school.html.erb

```diff
@@ -11,9 +11,9 @@
     <meta name="color-scheme" content="light">
     <meta name="supported-color-schemes" content="light">
   </head>
-  <body width="100%" style="margin: 0; padding: 8px;">
-    <center role="article" aria-roledescription="email" lang="en" style="width: 100%;">
-      <table style="font-family: Arial, sans-serif; width: 100%; text-align: left; max-width:768px; margin:auto; background: #fafafa; padding:0px 16px;">
+  <body width="100%" style="margin: 0; padding: 0px;">
```

Comment:
Do we need to specify zero padding and margin?
```suggestion
  <body width="100%">
```

Reply:
In major browsers, the <body> element has an 8px margin. To remove the default values we are using the reset values: `margin: 0px; padding: 0px;`